tests_v2/test_workspace_icon_actions.py

```
@@ -0,0 +1,114 @@
+        capture_output=True,
+        text=True,
+        check=False,
+        env=env,
+        timeout=5,
+    )
+    if proc.returncode != 0:
+        merged = f"{proc.stdout}\n{proc.stderr}".strip()
+        raise cmuxError(f"CLI failed ({' '.join(args)}): {merged}")
+    try:
+        return json.loads(proc.stdout or "{}")
+    except Exception as exc:
```

Ruff (0.15.12)

[warning] Do not catch blind exception: `Exception`

(BLE001)